### PR TITLE
tickets/DM-45524: bug fix and random field angles

### PR DESCRIPTION
**Pull request by jfcrenshaw**
Fixed bug and added random field angles in lsst.ts.wep.utils.modelUtils.forwardModelPair

### CODE REVIEW

File: python/lsst/ts/wep/utils/modelUtils.py [hunk 1]
```diff
@@ -88,11 +88,15 @@ def forwardModelPair(
...
+        only specified for the intra or extrafocal donut, both donuts use
+        that angle. If neither is specified, a random angle is used for
+        both. (the default is None)
     fieldAngleExtra : tuple, optional
-        Field angle, in degrees, of the extrafocal donut.
-        (the default is (0.23, -0.6))
+        Field angle, in degrees, of the extrafocal donut. If the angle is
+        only specified for the intra or extrafocal donut, both donuts use
+        that angle. If neither is specified, a random angle is used for
+        both. (the default is None)
```
Comment (jbkalmbach) [inline python/lsst/ts/wep/utils/modelUtils.py]: Should specify that it's the same random angle for both in the last case. It's not quite clear.

**Review by jbkalmbach — CHANGES_REQUESTED** [review]
This new function needs some tests so we don't miss any more things like this.

File: python/lsst/ts/wep/utils/modelUtils.py (hunk 1 above)
Comment (jfcrenshaw) [reply python/lsst/ts/wep/utils/modelUtils.py]: Done.

**Review by jbkalmbach — APPROVED** [review]
Looks good. Thanks for adding the tests.

**Pull request merged**